lib/trino-parquet/src/main/java/io/trino/parquet/reader/SimpleSliceInputStream.java

```diff
@@ -140,57 +139,27 @@ public int getByteArrayOffset()
         return offset + slice.byteArrayOffset();
     }
 
-    public void ensureBytesAvailable(int bytes)
-    {
-        checkPositionIndexes(offset, offset + bytes, slice.length());
-    }
-
-    /**
-     * Always check if needed data is available with ensureBytesAvailable method.
```

Comment:
Is the comment no longer viable? Maybe remove suffix then?

Reply:
suffix `unchecked` is copied from Slice